packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -74,26 +75,74 @@ contract L1StandardBridge_DepositETH_Test is Bridge_Initializer {
     // - ETH ends up in the optimismPortal
     function test_depositETH_succeeds() external {
         assertEq(address(op).balance, 0);
+        uint256 nonce = L1Messenger.messageNonce();
+        uint256 version = 0; // Internal constant in the OptimismPortal: DEPOSIT_VERSION
```

Comment:
This is internal? I feel like it shouldn't be

Reply:
Agreed - let's move to a separate pr/linear issue?